src/Components/ComparePublicationData/ComparePublicationDataModal.tsx

```diff
@@ -235,16 +244,34 @@ const ComparePublicationDataModal: FC<ComparePublicationDataModalProps> = ({
     }
   }
 
+  const successSnackBarActions = (key: any, resultId: string) => (
+    <>
+      <StyledSnackBarButton
+        startIcon={<LaunchIcon />}
+        rel="noopener noreferrer"
+        target="_blank"
+        href={`${CRISTIN_REACT_APP_URL}/results/show.jsf?id=${resultId}`}>
+        Vis publikasjon
+      </StyledSnackBarButton>
+      <StyledSnackBarButton
+        startIcon={<CancelIcon />}
+        onClick={() => {
+          closeSnackbar(key);
+        }}>
+        Lukk
+      </StyledSnackBarButton>
+    </>
+  );
+
   function handlePublicationImported(result: any) {
     setIsConfirmImportDialogOpen(false);
 
     if (result.status === 200) {
-      enqueueSnackbar(
-        'Importerte publikasjon (Cristin-id: ' + result.result.id + ' og tittel: ' + result.result.title + ')',
-        {
-          variant: 'success',
-        }
-      );
+      enqueueSnackbar('Importerte publikasjon ', {
```

Comment:
teksten var litt forvirrende for meg. hva med "Publikasjonen ble importert.", eller noe ?